rust/lance-encoding/src/encoder.rs

```diff
@@ -791,7 +791,7 @@ impl CompressionStrategy for CoreArrayEncodingStrategy {
         if let DataBlock::FixedWidth(ref fixed_width_data) = data {
             let bit_widths = data
                 .get_stat(Stat::BitWidth)
-                .expect("FixedWidthDataBlock should have valid bit width statistics");
+                .expect("FixedWidthDataBlock should have valid `Stat::BitWidth` statistics");
```

Comment:
Minor nit: if you find yourself repeating the same `expect` statement again and again then maybe it would be worth it to make an `expect_stat` method which does the `get_stat` / `expect` combination.

Reply:
ha, I actually didn't know about this, thanks for the suggestion. I will fill a separate PR for this.